examples/apeworx_everlasting_bots.py

```diff
@@ -470,7 +494,7 @@ def do_trade(trade):
     # Instantiate the config using the command line arguments as overrides.
     config = get_config()
     # Set up ape
-    provider = ape.networks.parse_network_choice("ethereum:local:foundry").__enter__()
+    provider = ape.networks.parse_network_choice("ethereum:local:foundry", provider_settings={"port": 8546})
```

Comment:
I don't think it's correct to remove `__enter__` as I mentioned in discord. Can you put it back in?

Also are you sure you want to hard-code the port like that? This means that the user has to be careful about their port selection when running anvil, right? If hardcoded port is necessary, can you update the readme to modify the anvil command to specify this port?

Reply:
it's hardcoded anyway, the default is 8545 and that conflicts with my local node on my desktop. will add a note in the README.

Reply:
oh I see you're thinking of people who run anvil locally, most likely on the default 8545

Reply:
this is only in an example, so not the main codebase. do we really need to mention it in the README?

Reply:
If someone is running anvil locally, they probably know how to manage port conflicts. Also it's not standard behavior, since we're not instructing them to do that. So I don't think we need to worry about non-standard behavior not working.

Reply:
I'll remove it since I don't need it on my laptop, and can just manually change it on my desktop